src/el_cl_launcher.star

```diff
@@ -163,6 +170,37 @@ def launch(
             sequencer_enabled,
             sequencer_context,
         )
+        all_el_contexts.append(el_context)
+
+        new_el_context = None
+        if "builder-op" in cl_service_name:
+            plan.print("Launching rollup-boost")
+
+            sequencer_el_context = all_el_contexts[0] # currently by default first EL is sequencer
+            plan.print(sequencer_el_context)
+            builder_el_context = get_builder_el_context(plan, all_el_contexts)
+            plan.print(builder_el_context)
+
+            rollup_boost_service = rollup_boost.launch(
```

Comment:
curious why are we launching it here instead of launching it as additional service?

Reply:
discussed offline (deploy sequence)